virttest/remote_commander/remote_master.py

```diff
@@ -214,6 +215,21 @@ def __init__(self, stdin, stdout, debug=False):
                                                   " not started.")
 
     def set_responder(self, responder):
+        """
+        :warning: Users's helper becomes a part of a remote_commander.
+        remote_commander is treated as a session. All sessions are part of
+        BaseVM object. If you provide unpickable helper then your BaseVM also
+        becames unpickable. You can catch something like::
```

Comment:
The travis failure is related to the warning block, which needs to be indented properly. So if you want to put the whole content into the warning it is suppose to look like this:

```rst
        :warning: Users's helper becomes a part of a remote_commander.
            remote_commander is treated as a session. All sessions are part of
            BaseVM object. If you provide unpickable helper then your BaseVM
            also becames unpickable. You can catch something like::

                File "/usr/lib64/python2.7/copy_reg.py", line 74, in _reduce_ex
                getstate = self.__getstate__
                KeyError: '__getstate__'

            For more info see::

                https://github.com/avocado-framework/avocado-vt/issues/455
```